packages/typegpu/src/data/std140.ts

```diff
@@ -45,6 +46,8 @@ export class SimpleTgpuData<TSchema extends AnySchema>
     this.expressionCode = code;
     this.size = this.measure(MaxValue).size;
   }
+  isLoose = false as const;
+  label?: string | undefined;
```

Comment:
Could we move these above the constructor? Just for readability and consistency.
Also, we can set `label` based on the `code` parameter passed into the constructor.
```suggestion
  public readonly isLoose = false as const;
  public readonly label: string;
```